src/common/utils.tsx

```
@@ -37,14 +37,16 @@ export const parseOptionId: (
-    options
-      .filter((option) => {
-        return id instanceof Array
-          ? id.map((i) => String(i)).includes(String(option.id))
-          : String(option.id) === String(id);
-      })
-      .map((option) => option.text),
+  return (
+    humanizeStrings(
+      options
+        .filter((option) => {
+          return id instanceof Array
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)